client/components/domains/domain-skip-suggestion/index.jsx

```diff
@@ -0,0 +1,53 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import DomainSuggestion from 'components/domains/domain-suggestion';
+
+class DomainSkipSuggestion extends React.Component {
+	static propTypes = {
+		onButtonClick: PropTypes.func.isRequired,
+	};
+
+	onButtonClick = () => {
+		this.props.onButtonClick( this.props.suggestion );
+	};
+
+	render() {
+		const { translate } = this.props;
+		const buttonContent = translate( 'Skip Purchase', {
+			context: 'Domain transfer or mapping suggestion button',
+		} );
+
+		return (
+			<DomainSuggestion
+				buttonContent={ buttonContent }
+				buttonStyles={ { borderless: true } }
+				extraClasses="is-visible domain-transfer-suggestion"
+				hidePrice={ true }
+				onButtonClick={ this.props.onButtonClick }
+				showChevron
+				// tracksButtonClickSource={ this.props.tracksButtonClickSource }
+			>
+				<div className="domain-transfer-suggestion__domain-description">
+					<h3>{ this.props.selectedSiteSlug }</h3>
+					<p>
+						{ translate( 'This is your current free site address', {
```

Comment:
:information_source: This string is probably long enough that it doesn't need a **context**. Are you sure you don't want to use a translator comment instead?